lab-jason/test/test-joke.js

```diff
@@ -0,0 +1,36 @@
+const assert = require('assert');
+
+const sut = require('../lib/joke');
+
+describe('joke', function(){
+  describe('#readJoke()', function(){
+    it(`should load txt files in following order : <one.txt>, <two.txt>, <three.txt>`, function(done){
+      sut.readJoke('one.txt', 'two.txt', 'three.txt', (err, data) =>{
+        assert.ifError(err);
+        assert.strictEqual(data, '486572652069732061486f77206d616e79204f6e652c2074686579');
```

Comment:
It looks like the code is working correctly, you just have an extra byte of expected data for each file.

```
      Uncaught AssertionError: '4865726520697320486f77206d616e794f6e652c20746865' === '486572652069732061486f77206d616e79204f6e652c2074686579'
      + expected - actual
      -4865726520697320486f77206d616e794f6e652c20746865
      +486572652069732061486f77206d616e79204f6e652c2074686579
```